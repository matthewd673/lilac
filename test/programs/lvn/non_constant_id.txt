i32 a = 1i32
i32 a = 5i32
i32 b = a * 2i32
i32 a = 3i32
i32 c = a + a
