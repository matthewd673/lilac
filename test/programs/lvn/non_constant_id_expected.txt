i32 a = 1i32
i32 a = 5i32
i32 b = 10i32
i32 a = 3i32
i32 c = 6i32
